src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -17,10 +17,16 @@
 public class LimelightSubsystem extends SubsystemBase {
 
     private final NetworkTable limelightTable;
+    private final Pose3d limelightOffsets;
 
-    public LimelightSubsystem() {
+    public LimelightSubsystem(Pose3d limelightOffsets) {
+        this.limelightOffsets = limelightOffsets;
         limelightTable = NetworkTableInstance.getDefault().getTable("limelight");
-        LimelightHelpers.setCameraPose_RobotSpace("limelight", Constants.LIMELIGHT_FORWARD_OFFSET , Constants.LIMELIGHT_SIDE_OFFSET, Constants.LIMELIGHT_UP_OFFSET, Constants.LIMELIGHT_ROLL_OFFSET, Constants.LIMELIGHT_PITCH_OFFSET, Constants.LIMELIGHT_YAW_OFFSET);
+        
+        LimelightHelpers.setCameraPose_RobotSpace("limelight", 
+        limelightOffsets.getX() , limelightOffsets.getY(), limelightOffsets.getZ(), 
+        Math.toDegrees(limelightOffsets.getRotation().getX()), Math.toDegrees(limelightOffsets.getRotation().getY()), 
```

Comment:
Same here, you could do `Angle.ofBaseUnits(limelightOffsets.getRotation().getX(), Units.Radians)`